include/nlohmann/detail/output/serializer.hpp

```diff
@@ -542,35 +576,78 @@ class serializer
                  int> = 0>
     void dump_integer(NumberType x)
     {
+        static constexpr std::array<std::array<char, 3>, 100> digits_to_99
```

Comment:
Why `std::array<char, 3>` and not `std::array<char, 2>`?

Reply:
Nice catch! I was experimenting with a larger lookup table (with 3x1000 entries). Forgot to revert it back to 2x100.